### PR TITLE
[CloudPC] Add property resetEnabled into CloudPCUserSetting

### PR DESCRIPTION
**Pull request by Guoan-Tang**
roadmap workitem:18059

### CODE REVIEW

**Comment by AshleyYangSZ** [comment]
@Guoan-Tang please add change-log in this pr.

File: api-reference/beta/api/cloudpcusersetting-update.md [hunk 1]
```diff
@@ -53,9 +53,11 @@ The following table shows the properties that are required when you update the [
...
+|resetEnabled|Boolean|Indicates whether an end-user is allowed to reset their Cloud PC. When `true`, the user is allowed to reset their Cloud PC. When `false`, end-user initiated reset is not allowed. The default value is `false`. |
 |selfServiceEnabled|Boolean|To turn on the self-service option, change this setting to `true`. |
 |restorePointSetting|[cloudPcRestorePointSetting](../resources/cloudpcrestorepointsetting.md)|Defines how frequently a restore point is created (that is, a snapshot is taken) for users' provisioned Cloud PCs (default is 12 hours), and whether the user is allowed to restore their own Cloud PCs to a backup made at a specific point in time.|
 |lastModifiedDateTime|DateTimeOffset|The last date and time the setting was modified. The Timestamp type represents the date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. |
+|resetEnabled|Boolean|Indicates whether an end-user is allowed to reset their Cloud PC. When `true`, the user is allowed to reset their Cloud PC. When `false`, end-user initiated reset is not allowed. The default value is `false`.|
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/cloudpcusersetting-update.md:60]: Why add duplicate line for resetEnabled? 
Could you help me understand what's reset mean? Is it reprovision?

File: api-reference/beta/api/cloudpcusersetting-update.md [hunk 2]
```diff
@@ -83,11 +85,13 @@ Content-Type: application/json
   "@odata.type": "#microsoft.graph.cloudPcUserSetting",
   "displayName": "Example",
   "selfServiceEnabled": true,
+  "resetEnabled": "Boolean",
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/cloudpcusersetting-update.md:88]: Please remove this

File: api-reference/beta/resources/cloudpcusersetting.md [hunk 3]
```diff
@@ -33,8 +33,10 @@ Represents a Cloud PC user setting.
...
+|resetEnabled|Boolean|Indicates whether an end-user is allowed to reset their Cloud PC. When `true`, the user is allowed to reset their Cloud PC. When `false`, end-user initiated reset is not allowed. The default value is `false`. |
 |restorePointSetting|[cloudPcRestorePointSetting](../resources/cloudpcrestorepointsetting.md)|Defines how frequently a restore point is created that is, a snapshot is taken) for users' provisioned Cloud PCs (default is 12 hours), and whether the user is allowed to restore their own Cloud PCs to a backup made at a specific point in time.|
 |selfServiceEnabled|Boolean|Indicates whether the self-service option is enabled. Default value is `false`. To enable the self-service option, change the setting to `true`. If the self-service option is enabled, the end user is allowed to perform some self-service operations, such as upgrading the Cloud PC through the end user portal.|
+|resetEnabled|Boolean|Indicates whether an end-user is allowed to reset their Cloud PC. When `true`, the user is allowed to reset their Cloud PC. When `false`, end-user initiated reset is not allowed. The default value is `false`.|
```
Comment (AshleyYangSZ) [inline api-reference/beta/resources/cloudpcusersetting.md:39]: Duplicate?

File: api-reference/beta/resources/cloudpcusersetting.md [hunk 4]
```diff
@@ -58,9 +60,11 @@ The following is a JSON representation of the resource.
   "id": "String (identifier)",  
   "lastModifiedDateTime": "String (timestamp)",
   "localAdminEnabled": "Boolean",
+  "resetEnabled": "Boolean",
   "restorePointSetting": {
     "@odata.type": "microsoft.graph.cloudPcRestorePointSetting"
   },
-  "selfServiceEnabled": "Boolean"
+  "selfServiceEnabled": "Boolean",
+  "resetEnabled": "Boolean"
```
Comment (AshleyYangSZ) [inline api-reference/beta/resources/cloudpcusersetting.md:68]: remove this

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/cloudpcusersetting-update.md (hunk 1 above)
Comment (xiaofuwahh) [reply api-reference/beta/api/cloudpcusersetting-update.md:60]: will deleted soon, yes reset is reprovision, this property name is align to IWP

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 5]
```diff
@@ -158,6 +158,25 @@
...
+                "Description": "Added the `resetEnabled` property to the [cloudPcUserSetting](https://learn.microsoft.com/en-us/graph/api/resources/cloudpcusersetting?view=graph-rest-beta) resource.",
+                "Target": "cloudPcUserSetting"
+              }
+
+            ],
+            "Id": "a244d924-f4ee-4f19-9e2d-a5a900ba82e5",
+            "Cloud": "Prod",
+            "Version": "beta",
+            "CreatedDateTime": "2023-05-12T10:21:43.9533784Z",
+            "WorkloadArea": "Devices and apps",
```
Comment (RuiHou105) [inline changelog/Microsoft.CloudManagedDesktop.json:177]:
```suggestion
            "WorkloadArea": "Devices and app management",
```

**Comment by RuiHou105** [comment]
Please create roadmap workitem.

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
Please resolve comments left in this PR.

File: api-reference/beta/resources/cloudpcusersetting.md (hunk 3 above)
Comment (xiaofuwahh) [reply api-reference/beta/resources/cloudpcusersetting.md]: deleted, thanks

**Review by RuiHou105 — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**